kuma/search/models.py

```diff
@@ -128,6 +127,8 @@ def promote(self):
             index_objects.delay(mapping_type, [instance.id])
             rescheduled.append(label)
         self.outdated_objects.all().delete()
+        # Allow only a single index to be promoted.
+        Index.objects.all().update(promoted=False)
```

Comment:
@jezdez Any problem here? I think it helps lead us to using index aliases but realize it may go against your plan. There's more simplifying we can do if this is ok, like `get_current()` doesn't need to sort by `-created` any longer, just get the promoted index.